src/rno-g.h

```diff
@@ -174,6 +175,15 @@ int rno_g_header_dump(FILE*f, const rno_g_header_t * header);
 int rno_g_header_write(rno_g_file_handle_t handle, const rno_g_header_t * header);
 int rno_g_header_read(rno_g_file_handle_t handle, rno_g_header_t * header);
 
+typedef struct radiant_readout_delay_struct
```

Comment:
for padding reasons, it's probably better to have the uint32_t first (see e.g. https://interrupt.memfault.com/blog/c-struct-padding-initialization or http://www.catb.org/esr/structure-packing/)

Reply:
I reordered the members here and the waveform struct after the LT samples - might not be super useful